code/cfile/cfilecompression.cpp

```diff
@@ -121,8 +121,8 @@ void lz41_create_ci(CFILE* cf, int header)
 	auto fSize = fread(&cf->size, sizeof(int), 1, cf->fp);
 	auto fBsize = fread(&cf->compression_info.block_size, sizeof(int), 1, cf->fp);
 
-	#if !defined(NDEBUG)
 	Assertion(cf->compression_info.num_offsets > 0, "Invalid number of offsets, compressed file is possibly in the wrong format or corrupted.");
```

Comment:
In order for this not to show up on release builds, you'll have to switch to the Verification macro, which works on both release and debug.

Reply:
Yeah i was wrong about Assertions usage, there is no need to run that on release, thats no more important than anything else on CFILE. Can we leave it as it is?

Reply:
Absolutely